### ISSUE TITLE
trojan virus, any ideas welcome

### ISSUE THREAD

**Issue by darul75** [bpatra/ExcelDNAWixInstallerLM#21]
After building this .msi and putting it online, end user always get this message while downloading it.

I have been trying to fix this issues for days with not success.

![virus](https://user-images.githubusercontent.com/2134003/32493470-46c87952-c3be-11e7-8be9-3da443b2c2bb.png)

**Comment by bpatra** [comment]
Woaw !!!
I can guarantee you that I did not write harmful code in this sample. Or at least not intentionally^^.

We actually leverage the active setup feature which is an almost undocumented feature of Windows. It is possible that the antivirus detects this usage and flags our .msi as dangerous.

As I explained in the blog post the hive registry key management is really complex to handle addins. It is possible also that the antivirus detects these complex key management and then consider our .msi harmful.

Sorry for not being of much help here....
